python/tvm/relax/op/transform.py

```diff
@@ -27,6 +27,24 @@
 PrimExprLike = Union[int, PrimExpr]
 
 
+def _handle_shape(input_shape: Union[PrimExprLike, List[PrimExprLike], Tuple[PrimExprLike], Expr]) -> Expr:
```

Comment:
Thanks for extracting this common part! One tiny point is that, “handle” is too general here and maybe we need a better name. What do you think of `_convert_shape_to_expr`?